games-board/stockfish/stockfish-sf_8.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="A chess engine claimed to be strongest in the world"
+DESCRIPTION="Stockfish is a free UCI chess engine derived from Glaurung 2.1. It \
+is not a complete chess program and requires some UCI-compatible GUI (e.g. \
+XBoard with PolyGlot, eboard, Arena, Sigma Chess, Shredder, ChessPartner or \
+Fritz) in order to be used comfortably. Read the documentation for your GUI of \
+choice for information about how to use Stockfish with it."
+HOMEPAGE="https://www.stockfishchess.org/"
+COPYRIGHT="2004-2008 Tord Romstad (Glaurung author)
+	2008-2017 Marco Costalba, Joona Kiiski, Gary Linscott, Tord Romstad"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/official-stockfish/Stockfish/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="e2b77d801d8837c22c2391b8cae08622c1434a2bb5947ddb08b081e1ad35c76c"
+SOURCE_DIR="Stockfish-sf_8"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64 ?arm ?ppc"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+if [ "$targetArchitecture" != x86_gcc2 ]; then
+	commandBinDir=$binDir
+else
+	commandBinDir=$prefix/bin
+fi
+case "$effectiveTargetArchitecture" in
+	x86) commandArch=x86-32;;
+	x86_64) commandArch=x86-64;;
+	arm) commandArch=armv7;;
+	ppc) commandArch=ppc-32;;
+esac
+
+PROVIDES="
+	stockfish$secondaryArchSuffix = $portVersion
+	cmd:stockfish$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	cd src/
+	make build \
+		ARCH=$commandArch
+		COMP=gcc
+}
+
+INSTALL()
+{
+	cd src/
+	make install \
+		BINDIR=$commandBinDir
+}
+
+TEST()
+{
+	true # Sorry, this project has no tests!
```

Comment:
From my previous pr, @pulkomandy told me that i could do something like this for reasons
>This allows us to remember that the project has no support. It can be useful, as in next years GCI, we may have tasks about researching which recipes would benefit from having a TEST block added, and we need to remember which one were already checked.